store/sqlstore/upgrade_test.go

```diff
@@ -105,140 +103,142 @@ func TestSaveSchemaVersion(t *testing.T) {
 		})
 	})
 }
-func createChannelMemberWithLastViewAt(ss store.Store, channelId, userId string, lastViewAt int64) *model.ChannelMember {
-	m := model.ChannelMember{}
-	m.ChannelId = channelId
-	m.UserId = userId
-	m.LastViewedAt = lastViewAt
-	m.NotifyProps = model.GetDefaultChannelNotifyProps()
-	cm, _ := ss.Channel().SaveMember(&m)
-	return cm
-}
-func createPostWithTimestamp(ss store.Store, channelId, userId, rootId, parentId string, timestamp int64) *model.Post {
-	m := model.Post{}
-	m.CreateAt = timestamp
-	m.ChannelId = channelId
-	m.UserId = userId
-	m.RootId = rootId
-	m.ParentId = parentId
-	m.Message = "zz" + model.NewId() + "b"
-	p, _ := ss.Post().Save(&m)
-	return p
-}
 
-func createChannelWithLastPostAt(ss store.Store, teamId, creatorId string, lastPostAt, msgCount, rootCount int64) (*model.Channel, error) {
-	m := model.Channel{}
-	m.TeamId = teamId
-	m.TotalMsgCount = msgCount
-	m.TotalMsgCountRoot = rootCount
-	m.LastPostAt = lastPostAt
-	m.CreatorId = creatorId
-	m.DisplayName = "Name"
-	m.Name = "zz" + model.NewId() + "b"
-	m.Type = model.CHANNEL_OPEN
-	return ss.Channel().Save(&m, -1)
-}
+//func createChannelMemberWithLastViewAt(ss store.Store, channelId, userId string, lastViewAt int64) *model.ChannelMember {
+//	m := model.ChannelMember{}
+//	m.ChannelId = channelId
+//	m.UserId = userId
+//	m.LastViewedAt = lastViewAt
+//	m.NotifyProps = model.GetDefaultChannelNotifyProps()
+//	cm, _ := ss.Channel().SaveMember(&m)
+//	return cm
+//}
+//func createPostWithTimestamp(ss store.Store, channelId, userId, rootId, parentId string, timestamp int64) *model.Post {
+//	m := model.Post{}
+//	m.CreateAt = timestamp
+//	m.ChannelId = channelId
+//	m.UserId = userId
+//	m.RootId = rootId
+//	m.ParentId = parentId
+//	m.Message = "zz" + model.NewId() + "b"
+//	p, _ := ss.Post().Save(&m)
+//	return p
+//}
+//
+//func createChannelWithLastPostAt(ss store.Store, teamId, creatorId string, lastPostAt, msgCount, rootCount int64) (*model.Channel, error) {
+//	m := model.Channel{}
+//	m.TeamId = teamId
+//	m.TotalMsgCount = msgCount
+//	m.TotalMsgCountRoot = rootCount
+//	m.LastPostAt = lastPostAt
+//	m.CreatorId = creatorId
+//	m.DisplayName = "Name"
+//	m.Name = "zz" + model.NewId() + "b"
+//	m.Type = model.CHANNEL_OPEN
+//	return ss.Channel().Save(&m, -1)
+//}
+
 func TestMsgCountRootMigration(t *testing.T) {
 	t.Skip("Waiting for this tests to be fixed: https://mattermost.atlassian.net/browse/MM-34792")
```

Comment:
@jwilander @marianunez I've detected some problems with these tests and I tried to fix them but it takes more time than I have so given that is going to affect Master as well Eli is taking care of them [here](https://mattermost.atlassian.net/browse/MM-34792). I